final_prep/README.md

```diff
@@ -19,37 +19,37 @@ Some things to consider as you refactor include...
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- ![Screen Shot 2021-08-07 at 10 24 57 AM](https://user-images.githubusercontent.com/81737385/128607045-e91bbb43-80e0-4e22-9e70-883c7ac24f50.png)
+- ![Screen Shot 2021-08-07 at 10 25 32 AM](https://user-images.githubusercontent.com/81737385/128607050-a1b787da-9b0e-4a14-b81b-14eb20528a86.png)
+- ![Screen Shot 2021-08-07 at 10 25 43 AM](https://user-images.githubusercontent.com/81737385/128607060-cea0991c-0675-43d9-891f-cedfafe4b509.png)
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: https://gist.github.com/JCNapier/35a726bc1c96c2ad4aab9a96b49f6435
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] Add a link to your gist here: https://gist.github.com/JCNapier/81f3c3c2f31b29345cf75c646f9db727
```

Comment:
I like seeing plenty of space in your notes to come back and add details, or connect ideas. During Turing, I found that taking notes about syntax were useful at times, but I think using different colors is more valuable for making key points stand out, rather than used as syntax highlighting... I feel like I may get behind in my note taking if I were trying to go for that!